app/src/main/java/com/github/warnastrophy/core/model/util/HazardsRepository.kt

```
@@ -53,59 +56,83 @@ class HazardsRepository {
         return message
```

It would be better to throw an exception and let the Overviewmodel handle it when there is a network Error